### PR TITLE
Added OCM method to install RHODA operator from Dev Catalog

### PR DESCRIPTION
**Pull request by mrrajan**
Signed-off-by: Rajan Ravi <rravi@redhat.com>

This PR includes updates to OCM.py file to install RHODA operator with the given Build version and RHOBS token.

### CODE REVIEW

File: utils/scripts/ocm/ocm.py [hunk 1]
```diff
@@ -1405,6 +1437,28 @@ def oc_login_as_admin(self):
     )
     dbaaspolicy_parser.set_defaults(func=ocm_obj.update_dbaaspolicy_config)
 
+    # Argument Parser to install RHODA for given version from Dev Catalog
+    install_rhoda_dev_parser = subparsers.add_parser(
+        "install_rhoda_dev",
+        help=("To Install RHODA from Develop Catalog Image"),
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
```
Comment (ikanias) [inline utils/scripts/ocm/ocm.py:1444]: What is this first section for?

File: utils/scripts/ocm/ocm.py [hunk 2]
```diff
@@ -1405,6 +1437,28 @@ def oc_login_as_admin(self):
...
+    )
+    required_install_rhoda_dev_parser = install_rhoda_dev_parser.add_argument_group("required arguments")
```
Comment (ikanias) [inline utils/scripts/ocm/ocm.py:1446]: Is this some kind of constructor for the class you created before this?

**Review by ikanias — APPROVED** [review]

File: utils/scripts/ocm/ocm.py (hunk 1 above)
Comment (mrrajan) [reply utils/scripts/ocm/ocm.py:1444]: @ikanias This is for Passing arguments to the method **_install_rhoda_dev_** specified on line 1460 **_install_rhoda_dev_parser.set_defaults(func=ocm_obj.install_rhoda_dev)_** 

We started with defining the argument parser to invoke method **_install_rhoda_dev_** on line 1441 to 1445. We are adding helper text and defining the class for the argument. So when we invoke ocm,py with the argument as **_./utils/scripts/ocm/ocm.py _install_rhoda_dev_**, the control passed here to verify the additional arguments required to invoke this method, in this case **--rhoda-version** and **--rhobs-token**

File: utils/scripts/ocm/ocm.py (hunk 2 above)
Comment (mrrajan) [reply utils/scripts/ocm/ocm.py:1446]: This is part of argument python argparse library. More details can be found on this doc - https://docs.python.org/3/howto/argparse.html

**Pull request merged**